### ISSUE TITLE
Allow custom prefix for the `logo` command

### ISSUE THREAD

**Issue by matthdsm** [nf-core/tools#2844]
### Description of feature

I know this is a long shot, but it'd be nice of the `logo` command would take into account the custom prefix from `.nf-core.yml` when creating a logo. Perhaps without the apple core, but it'd be cool to see all nf-core template based workflows to have a recognisable typefaced logo.

**Comment by mashehu** [comment]
hmm, that's a hard one, because we have the first line of the logo hardcoded: https://github.com/nf-core/tools/blob/master/nf_core/assets/logo/nf-core-repo-logo-base-darkbg.png (because generating pngs is hard)

**Comment by matthdsm** [comment]
I don't know how it's done now, but I imagine generating an SVG logo and then converting it to png?
Is this still done through the website API? I think the SVG approach could migrate the process entirely to tools?

**Comment by mashehu** [comment]
yes, it's part of tools now https://github.com/nf-core/tools/blob/master/nf_core/create_logo.py, and it would work for the svg approach, because just replace a string there, but unfortunately, svg->png is astonishingly difficult and requires some dependencies that are often a pain (cariosvg 👀 ). so at the moment, the svgs and the pngs are generated independent of each other.

**Comment by matthdsm** [comment]
I didn't know there was an svg option for the logo's! Is there an option to make the svg output customiseable so the user can do the conversion?

**Comment by mashehu** [comment]
I don't want to add too much functionality only to one output format, so you'll have to edit the svg by yourself I am afraid.

**Comment by matthdsm** [comment]
No problemo. Thanks for the info!
